### PR TITLE
Add missing attributes to the subscription model

### PR DESCRIPTION
**Pull request by blaix**

### CODE REVIEW

File: tests/models/test_subscription.py [hunk 1]
```diff
@@ -6,6 +7,19 @@
 
 
 class SubscriptionTestCase(TestCase):
+    fake_subscription_data = {
+        'id': 'fake-sub-id-abcdef1234567890abcd',
+        'auto_renew': False,
+        'billing_date': datetime.now(),
```
Comment (snitch) [inline tests/models/test_subscription.py]: S100 Snippet found: "datetime.now".

File: tests/models/test_subscription.py [hunk 2]
```diff
@@ -6,6 +7,19 @@
...
+        'deprovision_date': datetime.now(),
```
Comment (snitch) [inline tests/models/test_subscription.py]: S100 Snippet found: "datetime.now".

File: tests/models/test_subscription.py [hunk 3]
```diff
@@ -6,6 +7,19 @@
...
+        'expiry_date': datetime.now(),
```
Comment (snitch) [inline tests/models/test_subscription.py]: S100 Snippet found: "datetime.now".

File: tests/models/test_subscription.py [hunk 4]
```diff
@@ -6,6 +7,19 @@
...
+        'start_date': datetime.now(),
```
Comment (snitch) [inline tests/models/test_subscription.py]: S100 Snippet found: "datetime.now".

File: tests/models/test_subscription.py (hunk 2 above)
Comment (blaix) [inline tests/models/test_subscription.py]: wat?

Comment (adrianmoisey) [inline tests/models/test_subscription.py]: He wants `datetime.utnow()`

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by blaix** [comment]
Fixed snitch's complaints and the travis failures.

**Comment by blaix** [comment]
ping @beck for review

**Comment by beck** [comment]
:+1:

**Pull request merged**